### DISCUSSION TITLE
No Snips on Blink (Classic Plan)

### DISCUSSION THREAD

**Discussion by mccurdyc** [blinksh/blink#2074] [Support] [answered]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

- iPadOS 18.1
- Blink v18.0.1.940
   - iCloud Sync > On
   - XCallBack URL > On
- Blink Subscription Classic Plan (1-time purchase)
   - Do I need to "Upgrade" to the monthly plan?

### Describe the bug

- expected to see a "snips" icon per these docs - https://github.com/blinksh/docs/blob/master/src/pages/basics/snips.md / https://docs.blink.sh/basics/snips

<img width="1600" alt="image" src="https://github.com/user-attachments/assets/d8980df4-3843-4439-ab9c-a7d909d162d1">

<img width="1600" alt="image" src="https://github.com/user-attachments/assets/de67ca4b-3877-4d38-9838-3cb17415b338">

<img width="1600" alt="image" src="https://github.com/user-attachments/assets/4e4e6596-55bc-4328-bfb2-9c211e54f546">

**Comment by carloscabanero** [comment, marked as answer]
Snips are a Blink Plus feature, and it is not included within Blink Classic. Blink Classic has all the features from the previous Blink, and all the fixes and updates required to keep the app working in an on-going basis. You do not need the monthly version for just Blink Plus, you can subscribe to the annual version for $20/year ($1.67/month).

**Reply by mccurdyc** [reply]
Makes sense! That’s what I figured, but I didn’t see it explained.

Thank you so much! I love Blink!
